src/segmented-control/__tests__/desktop.test.tsx

```diff
@@ -141,4 +142,18 @@ describe('selected property', () => {
     getSegmentWrapper(segmentedControlWrapper, 0).getElement().click();
     expect(onChange).toHaveBeenCalledWith(expect.objectContaining({ detail: { selectedId: 'seg-1' } }));
   });
+
+  test('does change focus when left/right arrow key is pressed', () => {
+    const { segmentedControlWrapper } = renderSegmentedControl(
+      <SegmentedControl selectedId="seg-1" options={defaultOptions} />
+    );
+
+    getSegmentWrapper(segmentedControlWrapper, 0).getElement().focus();
+    getSegmentWrapper(segmentedControlWrapper, 0).keydown(KeyCode.right);
+    expect(document.activeElement).toHaveAttribute('aria-label', 'Icon for Segment-2');
```

Comment:
This assertion should be 

```js
expect(getSegmentWrapper(segmentedControlWrapper, 1)).toHaveFocus()
```

And similar for all other assertions here

Reply:
Good catch, updated.